napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -92,7 +92,11 @@ def __init__(self, layer):
         )
         sld.setFocusPolicy(Qt.FocusPolicy.NoFocus)
         sld.setMinimum(1)
-        sld.setMaximum(100)
+        try:
+            max_value = max(100, int(np.max(self.layer.size)) + 1)
+        except ValueError:
+            max_value = 100
```

Comment:
```suggestion
        if self.layer.size.size:
            max_value = max(100, int(np.max(self.layer.size)) + 1)
        else:
            max_value = 100
```
As an empty list of points is common to be a constructor argument I prefer `if` instead of an exception.

Reply:
Agreed, changed now.